src/Repository/GeoFilterTrait.php

```diff
@@ -26,6 +26,15 @@ private function applyDeputyGeoFilter(QueryBuilder $qb, Adherent $deputy, string
         $this->applyGeoFilter($qb, [$deputy->getManagedDistrict()->getReferentTag()], $alias);
     }
 
+    private function applySenatorGeoFilter(QueryBuilder $qb, Adherent $senator, string $alias): void
```

Comment:
it seems to be unused

Reply:
wow,  good catch ! I indeed forgot to use it ! Fixed